src/RecoveryServices/RecoveryServices/help/Set-AzRecoveryServicesAsrReplicationProtectedItem.md

```diff
@@ -501,6 +549,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SqlServerLicenseType
+Specify the SQL Server license type of the VM.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+Accepted values: NoLicenseType, PAYG, AHUB
```

Comment:
>NoLicenseType [](start = 17, length = 13)

Same as above

Reply:
Will cross check on this and will add in next PR if any change required.

---
In reply to: [601541382](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601541382) [](ancestors = 601541382)